packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -8,18 +8,13 @@
         {
             "name": "documentFormat",
             "type": "string",
-            "allowedValues": [
-                "JSON, YAML"
-            ],
+            "allowedValues": ["JSON, YAML"],
```

Comment:
Is there some autoformatting be applied after merges or something? If you format the vscodeTelemetry.json file it shows as the left but when you commit it it shows like the right

I'd guess its husky doing something. Its a little confusing why this wants to show like its formatted here, but once we merge this PR it will format like the left again

Reply:
do you have a eslint plugin in vscode? that can do random things (and isn't really needed).